### DISCUSSION TITLE
connecting to nightscout from AAPS

### DISCUSSION THREAD

**Discussion by AlJeske** [NightscoutFoundation/xDrip#3257] [General]
Trouble connecting nightscout to AAPS
![Screenshot_20231224-093406](https://github.com/NightscoutFoundation/xDrip/assets/50727751/aee2fcc9-59e4-4030-93f5-b6b39a1e119f)
![Screenshot_20231222-230459](https://github.com/NightscoutFoundation/xDrip/assets/50727751/66579c34-784b-49c9-9b7f-ac17c33b4775)
Using the latest version of AAPS.

**Comment by Navid200** [comment]
Is there no AAPS support?
Is there no forum?

**Reply by AlJeske** [reply]
Wrong place to look on this forum. ? I'll check with AAPS.

**Reply by Navid200** [reply]
It's not wrong.
But, I am making sure you know that this is not an AAPS forum.

I think you should ask in a relevant forum before posting in a sister forum.
